### DISCUSSION TITLE
library import issue

### DISCUSSION THREAD

**Discussion by DonnaShin** [lululxvi/deepxde#1366] [Q&A]
Hello, LuLu

I installed deepxde in a new virtual environment (tensorflow-gpu 2.8.0), but an error occurs.
I'm not sure how to solve it, even looking at the FAQ or backend explanation.
The error is something like this:
------------------------------------------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[5], line 4
      1 import numpy as np
      2 from scipy.integrate import odeint
----> 4 import deepxde as dde
      5 from deepxde.backend import tf

.....

File ~\anaconda3\envs\xde2\lib\site-packages\tensorflow\python\util\lazy_loader.py:62, in LazyLoader.__getattr__(self, item)
     61 def __getattr__(self, item):
---> 62   module = self._load()
     63   return getattr(module, item)

File ~\anaconda3\envs\xde2\lib\site-packages\tensorflow\python\util\lazy_loader.py:45, in LazyLoader._load(self)
     43 """Load the module and insert it into the parent's globals."""
     44 # Import the target module and insert it into the parent's namespace
---> 45 module = importlib.import_module(self.__name__)
     46 self._parent_module_globals[self._local_name] = module
     48 # Emit a warning if one was specified

File ~\anaconda3\envs\xde2\lib\importlib\__init__.py:127, in import_module(name, package)
    125             break
    126         level += 1
--> 127 return _bootstrap._gcd_import(name[level:], package, level)

ModuleNotFoundError: No module named 'keras'

------------------------------------------------------------------------------------------------------------

How do I solve this?

**Comment by lululxvi** [comment]
> ModuleNotFoundError: No module named 'keras'

Is your tensorflow installed correctly?
